package.json

```diff
@@ -82,29 +82,29 @@
 		"@types/jsdom": "^16.2.3",
 		"@types/mini-css-extract-plugin": "^0.9.1",
 		"@types/react": "^16.9.46",
-		"@types/terser-webpack-plugin": "^4.0.0",
-		"ava": "^3.11.1",
+		"@types/terser-webpack-plugin": "^4.1.0",
+		"ava": "^3.12.1",
 		"chrome-webstore-upload-cli": "^1.2.0",
 		"copy-webpack-plugin": "^6.0.3",
 		"css-loader": "^4.2.1",
 		"daily-version": "^2.0.0",
 		"dot-json": "^1.2.0",
 		"eslint-config-xo-react": "^0.23.0",
-		"eslint-plugin-react": "^7.20.5",
-		"eslint-plugin-react-hooks": "^4.0.8",
+		"eslint-plugin-react": "^7.20.6",
+		"eslint-plugin-react-hooks": "^4.1.0",
 		"jsdom": "^16.4.0",
-		"mini-css-extract-plugin": "^0.9.0",
+		"mini-css-extract-plugin": "^0.10.0",
 		"npm-run-all": "^4.1.5",
-		"rollup": "^2.23.1",
+		"rollup": "^2.26.5",
 		"size-plugin": "^2.0.1",
 		"string-replace-loader": "^2.3.0",
 		"stylelint": "^13.6.1",
 		"stylelint-config-xo": "^0.19.0",
 		"terser-webpack-plugin": "^4.1.0",
-		"ts-loader": "^8.0.2",
-		"ts-node": "^8.10.2",
+		"ts-loader": "^8.0.3",
+		"ts-node": "^9.0.0",
 		"type-fest": "^0.16.0",
-		"typescript": "^4.1.0-dev.20200810",
+		"typescript": "^4.0.2",
```

Comment:
We're going back? I want to make sure that the reason why this was added has been solved in v4. Tonight I have no data left though

Reply:
I checked I get no messages. I put it on the latest stable version. want me to check the latest dev version?

> Tonight I have no data left though

You run on limited data?? -  Wow

Reply:
> You run on limited data?? - Wow

WiFi wasn't working and promptly ran out of 4G data 🤣

Reply:
Tested as well, it works